src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetalingAndelSkjema.tsx

```diff
@@ -179,7 +179,7 @@ const EndretUtbetalingAndelSkjema: React.FunctionComponent<IEndretUtbetalingAnde
                     <StyledPersonvelger
                         {...skjema.felter.person.hentNavBaseSkjemaProps(skjema.visFeilmeldinger)}
                         label={<Label>Velg hvem det gjelder</Label>}
-                        value={skjema.felter.person.verdi}
+                        value={skjema.felter.person.verdi ?? ''}
```

Comment:
Fikk denne feilmelding i console før: `value` prop on `select` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components